parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -15,6 +15,17 @@
 // limitations under the License.
 -->
 
+/* Extra operators */
+
+<DEFAULT, DQID, BTID> TOKEN :
```

Comment:
@AndrewPochapsky will our DialectGenerate code handle this syntax in an .ftl file?

Reply:
I don't think that it will in its current state as it is of a different format than the regular functions.

Reply:
If we can generalize statements like this then I can come up with a parsing approach.

Reply:
We could also move this inside of Parser.jj and conditionally template it in with a variable since this is an outlier unless we wanted to add support for this inside of ftl files

Reply:
Andrew and I spoke about this and he has a plan to handle extracting these clauses (TOKEN, SKIP, and MORE) from .ftl files.